Changes

```diff
@@ -2,6 +2,9 @@ Revision history for App-AizuOnlineJudge
 
 {{$NEXT}}
 
+0.0.3  2019-08-19T11:11:00+01:00
```

Comment:
`{{$NEXT}}` is the placeholder for `0.0.3`, so could you write as follows?:
```
{{$NEXT}}
    - Fix compatibility with cro 0.8
```

Reply:
Ah, thank you, had no idea how the format works and tried to guess. :)